clients/go/pkg/commands/stream_jobs.go

```diff
@@ -104,21 +104,20 @@ func (cmd *StreamJobsCommand) Send(ctx context.Context) error {
 
 	for {
 		job, err := stream.Recv()
-		if err == io.EOF {
-			break
-		}
+		if err != nil {
+			if err == io.EOF {
+				break
```

Comment:
Just to clarify: EOF is an acceptable error which is why we don't return it?

Reply:
Yeah, it's the equivalent of `StreamObserver#onCompleted` in Java.